libclingcon/src/clingcon.cc

```diff
@@ -379,7 +379,7 @@ extern "C" bool clingcon_destroy(clingcon_theory_t *theory) {
 
 extern "C" bool clingcon_configure(clingcon_theory_t *theory, char const *key, char const *value) {
     CLINGCON_TRY {
-        auto config = theory->propagator.config();
+        auto& config = theory->propagator.config();
```

Comment:
Can you change the space to be consistent with my indentation style. The CHANGES.md should also mention bugfixes.